controllers/datamover.go

```diff
@@ -673,6 +727,12 @@ func (r *DPAReconciler) ReconcileDataMoverResticSecret(log logr.Logger) (bool, e
 				if err != nil {
 					return false, err
 				}
+
+				_, err = r.createResticCustomCASecret(&dpa, bsl)
```

Comment:
We may want a separate reconcile func for this secret so we do not have multiple objects being reconciled on in one

Reply:
I moved this to its own function: https://github.com/openshift/oadp-operator/pull/918/commits/1958bbebcde1443a4fbba428a201ea07086fadf2